build.sbt

```diff
@@ -23,8 +23,8 @@ ThisBuild / organization := "at.forsyte"
 ThisBuild / scalaVersion := "2.13.12"
 
 // Add resolver for Sonatype OSS Snapshots and Releases Maven repository
-ThisBuild / resolvers ++= Resolver.sonatypeOssRepos("snapshots")
-ThisBuild / resolvers ++= Resolver.sonatypeOssRepos("releases")
+ThisBuild / resolvers += Resolver.sonatypeRepo("snapshots")
+ThisBuild / resolvers += Resolver.sonatypeRepo("releases")
 
```

Comment:
I remember `sonatypeRepo` triggering deprecation warnings – I upgraded to `sonatypeOssRepos` in #2612. Is it safe to go back?

Reply:
removed this